### PR TITLE
[build.yaml] scale dev namespaces to zero 9p-9a weekdays and weekends

### PR DESCRIPTION
**Pull request by danking**
cc: @jigold @iris-garden @sjparsa , since you all use dev namespaces.

### CODE REVIEW

File: build.yaml [hunk 1]
```diff
@@ -3679,3 +3679,73 @@ steps:
...
+      set -ex
+      {% for user in code.get("developers", []) %}
+      {% if user['username'] != 'test-dev' %}
+      apiVersion: batch/v1
+      kind: CronJob
+      metadata:
+        name: dev-namespace-scaledown-{{ user["username"] }}
+        namespace: {{ user["username"] }}
+      spec:
+        schedule: 0 22 * * 1,2,3,4,5"  # Weekdays at 8p
```
Comment (daniel-goldstein) [inline build.yaml:3699]: 10p, no?

**Review by daniel-goldstein — CHANGES_REQUESTED** [review]
This is a great and easy change

File: build.yaml [hunk 2]
```diff
@@ -3679,3 +3679,73 @@ steps:
...
+              spec:
+                serviceAccountName: admin
+                containers:
+                - name: dev-namespace-daytime-autoscaler
+                  image: bitnami/kubectl:latest
+                  command:
+                  - /bin/sh
+                  - -c
+                  - set -ex ; kubectl scale deployments --all -n {{ user["username"] }} --replicas=1 && kubectl scale statefulsets --all -n {{ user["username"] }} --replicas=1
+                restartPolicy: OnFailure
```
Comment (daniel-goldstein) [inline build.yaml:3739]: Don't you need a `---` delimeter at the end too since this is in a loop?

File: build.yaml (hunk 1 above)
Comment (danking) [reply build.yaml:3699]: Heh, yeah, I was experimenting with it at 10p. Fixed.

File: build.yaml (hunk 2 above)
Comment (danking) [reply build.yaml:3740]: yep.

**Review by daniel-goldstein — APPROVED** [review]

**Pull request merged**